ow_gazebo_plugins/src/BalovnevModelPlugin/BalovnevModelPlugin.cpp

```diff
@@ -31,7 +31,7 @@ const double BUCKET_WIDTH             = 0.085;         // m
 const double BUCKET_SIDE_PLATE_LENGTH = 0.1225;        // m
 const double BUCKET_HEIGHT_FROM_TIP   = 0.0026;        // m
 const double BLUNT_EDGE_THICK         = 0.0004;        // m
-const double BLUNT_EDGE_ANGLE         = 89 * DEG2RAD;  // rad
+const double BLUNT_EDGE_ANGLE         = 30 * DEG2RAD;  // rad
```

Comment:
Would be good to have a comment describing what this constant means. I can't figure it out from the name.

Reply:
It would take a lot of paragraph length comments, but I agree. I would like to do that for each of these constants. Made a ticket for this: [OW-1211](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-1211)

This happens to be one of the values I never got a solid answer about from the literature I found, but believe it to be this angle on the scoop here. 

![Screenshot_20230914_132317](https://github.com/nasa/ow_simulator/assets/17039973/bbcd4174-7bd0-4e33-ae35-b1076f811b0f)
![Screenshot_20230914_132344](https://github.com/nasa/ow_simulator/assets/17039973/ab8b6074-3baa-463c-a227-b6b1f523640e)